.github/workflows/consume-release.yml

```
@@ -6,13 +6,40 @@ on:
   workflow_run:
     workflows: [Release]
     types: [completed]
+    branches: [develop]
 
 jobs:
-  on-success:
+  get-images:
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

```suggestion
    if:  github.event.workflow_run.conclusion == 'success' 
```

You can omit the expression evaluation syntax here, GHA treats `if` statements as expressions by default :) https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idif